Dockerfiles/agent/windows/entrypoint/entrypoint.cpp

```diff
@@ -66,7 +66,9 @@ void ExecuteInitScripts()
         DWORD exitCode = pwsh.WaitForExit();
         if (exitCode != 0)
         {
-            std::cout << "[ENTRYPOINT][WARNING] " << script.path() << " exited with code [" << std::hex << exitCode << "]" << std::endl;
+            std::stringstream sstream;
+            sstream << script.path() << " exited with code [" << std::hex << exitCode << "]";
```

Comment:
Nit: could we reuse `FormatErrorCode` defined above, for the `"[" << std::hex << exitCode << "]"` part?